SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/ReferenceCountedLockingPolicy.h

```diff
@@ -19,35 +21,47 @@ class SPATIALGDK_API UReferenceCountedLockingPolicy : public UAbstractLockingPol
 public:
 	virtual ActorLockToken AcquireLock(AActor* Actor, FString DebugString = "") override;
 
-	// This should only be called during the lifetime of the locked actor
+	// This should only be called during the lifetime of the locked Actor
 	virtual bool ReleaseLock(const ActorLockToken Token) override;
 
 	virtual bool IsLocked(const AActor* Actor) const override;
 
+	virtual void OnOwnerUpdated(AActor* Actor) override;
+
 private:
 	struct MigrationLockElement
 	{
 		int32 LockCount;
-		TFunction<void()> UnbindActorDeletionDelegateFunc;
```

Comment:
Previously we store a lambda here that's triggered when an Actor is deleted so we can clean up its locks. I don't think this is necessary.